spec.md

```diff
@@ -290,3 +293,24 @@ The following example shows a CloudEvent serialized as JSON:
     "data" : "<much wow=\"xml\"/>"
 }
 ```
+The following example shows a CloudEvent with JSON as `data` and relative URI in `source`:
```

Comment:
In the spirit of keeping PRs as small as possible, I'd suggest to remove the second example (and rather open up a follow-on PR).

Otherwise, this PR mixes two topics:
* Should we enforce the URI scheme, or should we loosen it up to URI-references?
* What is a good second example?

These aren't really related. Personally, I think the second example should differ more from the first one - but it is a bit off-topic.

Reply:
Right, understood!

I will keep this PR small and remove the second example.